app/controllers/events/view/videoroom/list.js

```diff
@@ -22,52 +25,78 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
   get columns() {
     return [
       {
-        name            : this.l10n.t('Microlocation'),
-        valuePath       : 'name',
-        helperInfo      : this.l10n.t('You need to add rooms into the wizard first before they show up here.'),
+        name       : this.l10n.t('Microlocation'),
+        valuePath  : 'name',
+        helperInfo : this.l10n.t(
+          'You need to add rooms into the wizard first before they show up here.'
+        ),
         extraValuePaths : ['id', 'videoStream', 'constructor'],
         cellComponent   : 'ui-table/cell/events/view/videoroom/cell-stream-title',
-        width           : 70,
+        width           : 55,
         actions         : {
           delete: this.delete.bind(this)
         }
       },
       {
         name       : this.l10n.t('Video Room Name'),
         valuePath  : 'videoStream.name',
-        helperInfo : this.l10n.t('This column shows the video room name that will be visible to the users on the event page.'),
-        width      : 70
+        helperInfo : this.l10n.t(
+          'This column shows the video room name that will be visible to the users on the event page.'
+        ),
+        width: 70
       },
       {
-        name          : this.l10n.t('Video Source URL'),
-        valuePath     : 'videoStream',
-        helperInfo    : this.l10n.t('This column shows the original link of the video. We do not recommend to share this link as users can access it without logging into') + this.settings.appName + '.com.',
+        name       : this.l10n.t('Video Source URL'),
+        valuePath  : 'videoStream',
+        helperInfo :
+          this.l10n.t(
+            'This column shows the original link of the video. We do not recommend to share this link as users can access it without logging into'
```

Comment:
why these kind of changes?

Reply:
Pre-commit hook

Reply:
Also, the hook follows the standard ES7 rules for writing code.